src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
+  const longRent = 7;
+  const longRentDiscount = 50;
+
+  const shortRent = 3;
+  const shortRentDiscount = 20;
+
+  if (days >= longRent) {
+    return (days * dayRentCost) - longRentDiscount;
```

Comment:
```suggestion
    return basePrice - LONG_TERM_DISCOUNT;
```
<img width="398" alt="Screenshot 2023-12-14 at 15 49 06" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/59fa93ec-79de-49f4-b085-e9b353320c98">